helper/utilty.py

```diff
@@ -517,10 +530,19 @@ def compute_psnr_and_ssim(image1, image2, border_size=0):
     if border_size > 0:
         image1 = image1[border_size:-border_size, border_size:-border_size, :]
         image2 = image2[border_size:-border_size, border_size:-border_size, :]
+    
 
-    psnr = compare_psnr(image1, image2, data_range=255)
-    ssim = compare_ssim(image1, image2, win_size=11, gaussian_weights=True, multichannel=True, K1=0.01, K2=0.03,
-                        sigma=1.5, data_range=255)
+    try:
+        psnr = compare_psnr(image1, image2, data_range=255)
+        ssim = compare_ssim(image1, image2, win_size=11, gaussian_weights=True, multichannel=True, K1=0.01, K2=0.03,
+                            sigma=1.5, data_range=255)
+    except ValueError:
```

Comment:
For this case, I guess we should check numpy array type before  compare_psnr() and branch it if the type is not suitable?